### PR TITLE
added a flag --skip-id-set-creation for validate command

### PR DESCRIPTION
**Pull request by abaumgarten**
## Status
- [] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Description
Added a flag --skip-id-set-creation for validate
Added the capability to run validate command without creating id_set validation.

### CODE REVIEW

**Comment by abaumgarten** [comment]
Build is failing on the error from here: demisto/etc#31320

File: demisto_sdk/commands/validate/validate_manager.py [hunk 1]
```diff
@@ -1170,7 +1172,7 @@ def get_id_set_file(skip_dependencies, id_set_path):
         """
         id_set = {}
         if not os.path.isfile(id_set_path):
-            if not skip_dependencies:
+            if not skip_id_set_creation:
```
Comment (guykeller) [inline demisto_sdk/commands/validate/validate_manager.py:1175]: 1. Let's make this one condition
2. Still not seeing why this is needed - when do we not want to create an ID set, given no path to an existing ID set has been specified?

Comment (roysagi) [reply demisto_sdk/commands/validate/validate_manager.py:1175]: @guykeller - some users do not need the id_set validation also this is in use in some test cases. This is an extra feature for this command so the capability to turn it off is a good option.

**Review by roysagi — APPROVED** [review]
@abaumgarten - please check that this change makes the script id validation for playbooks run even when running validate with `-a`

File: demisto_sdk/commands/validate/validate_manager.py (hunk 1 above)
Comment (guykeller) [reply demisto_sdk/commands/validate/validate_manager.py:1175]: So let's add that to the PR, so people will be able to know why it was created

**Pull request merged**